tap_twitter_ads/__init__.py

```diff
@@ -21,8 +22,9 @@
     'account_ids'
 ]
 
-def do_discover(reports):
+def do_discover(reports, client):
     LOGGER.info('Starting discover')
+    get_resource("test", client, 'accounts') # checking credentials for the discover mode
```

Comment:
As we are using the accounts endpoint. Can we write like below?
```suggestion
    get_resource("accounts", client, 'accounts') # checking credentials for the discover mode
```
```suggestion
    get_resource("test", client, 'accounts') # checking credentials for the discover mode
```

Reply:
Updated